Source/WebCore/page/LocalFrameViewLayoutContext.cpp

```diff
@@ -660,6 +660,29 @@ void LocalFrameViewLayoutContext::popLayoutState()
     }
 }
 
+void LocalFrameViewLayoutContext::markNeedingTransformUpdate(RenderLayerModelObject* parent, RenderLayerModelObject* descendant)
+{
+    if (m_descendantsNeedingTransformUpdate.contains(parent)) {
+        auto descendants = m_descendantsNeedingTransformUpdate.get(parent);
+        if (!descendants.contains(descendant))
+            m_descendantsNeedingTransformUpdate.get(parent).append(descendant);
+    } else
+        m_descendantsNeedingTransformUpdate.set(parent, Vector({ WeakPtr { descendant } }));
+}
+
+Vector<WeakPtr<RenderLayerModelObject>> LocalFrameViewLayoutContext::descendantsNeedingTransformUpdate(RenderLayerModelObject* parent)
+{
+    if (m_descendantsNeedingTransformUpdate.contains(parent))
```

Comment:
This is two hash lookups. You could do it in one with `find()`, or maybe even `get()` does the right thing if there's no matching entry.

Reply:
Fixed.